client/js/verification-portal/form.js

```diff
@@ -270,7 +302,7 @@ class VerificationPortalForm extends Component {
 
 				<div className={classes.parent}>
 					<Field
-						label="Hours Verified"
+						label="Verified"
```

Comment:
We can remove this label. The "Verified" label is only listed once, as a heading at the top, on the new designs.